pulp/pulp.py

```diff
@@ -59,7 +59,7 @@
 
 Display the status of the solution
 >>> const.LpStatus[status]
-'Optimal'
+'Optimal Solution Found'
```

Comment:
I think this line shouldn't change.

Reply:
@pchtsp I corrected the pulp.py, must have made an auto-change